guides/v2.1/extension-dev-guide/versioning/codebase-changes.md

```diff
@@ -62,6 +62,7 @@ public function save()
 
 This table lists code change scenarios and which version number it affects.
 Use this table to understand what changes Magento can make and which version number gets increased for that change.
+Any change not listed below is considered a PATCH level change.
```

Comment:
Has this list been checked against our semver tool's rules (including Magento-custom rules and semverchecker's defaults)?

Reply:
This document should be a base for SemVer checker. A team is currently working on improving the tool and they are checking it with the document.
cc @irenelagno